pkg/runs/runsGet.go

```diff
@@ -99,6 +105,31 @@ func validateOutputFormatFlagValue(outputFormatString string, validFormatters ma
 	return chosenFormatter, err
 }
 
+func GetRunDetailsFromRasSearchRuns(runs []galasaapi.Run, apiServerUrl string) ([]galasaapi.Run, error) {
+	var err error = nil
+	var runsDetails []galasaapi.Run = make([]galasaapi.Run, 0)
+	restClient := api.InitialiseAPI(apiServerUrl)
+	var context context.Context = nil
+	var details *galasaapi.Run
+	var httpResponse *http.Response
+
+	for _, run := range runs {
+		runid := run.GetRunId()
+		details, httpResponse, err = restClient.ResultArchiveStoreAPIApi.GetRasRunById(context, runid).Execute()
+		if err != nil {
+			err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_QUERY_RUNS_FAILED, err.Error())
+		} else {
+			if httpResponse.StatusCode != http.StatusOK {
+				err = galasaErrors.NewGalasaError(galasaErrors.GALASA_ERROR_QUERY_RUNS_FAILED, err.Error())
```

Comment:
Might be nice to log the http status code, or even mention it in the error message ?